Lib/test/test_statistics.py

```diff
@@ -1033,6 +1037,47 @@ def test_error_msg(self):
         self.assertEqual(errmsg, msg)
 
 
+class FindLteqTest(unittest.TestCase):
+    # Test _find_lteq private function.
+
+    def test_raise_value_error(self):
```

Comment:
Let's also find a better name for this, e.g. `test_invalid_input_values`

Reply:
Done.